dist/stat_data.py

```diff
@@ -586,6 +586,7 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_snapshots_created', 'number of named snapshots created'),
     TxnStat('txn_snapshots_dropped', 'number of named snapshots dropped'),
     TxnStat('txn_sync', 'transaction sync calls'),
+    TxnStat('txn_timestamp_oldest_active_read', 'transaction oldest active read timestamp', 'no_clear,no_scale'),
```

Comment:
I'd prefer to name this `txn_pinned_timestamp_reader`, and have the description similar to `txn_pinned_timestamp_oldest` ie:
`transaction range of timestamps pinned by the oldest active read timestamp`